rundeckapp/grails-app/services/rundeck/services/ProjectService.groovy

```diff
@@ -1947,6 +1954,111 @@ class ProjectService implements InitializingBean, ExecutionFileProducer, EventPu
         return result
     }
 
+    /**
+     * Creates the status file in db storage, the status file is the document that will have all the information
+     * about what is going on in the whole import operation. Will be requested in 'apiProjectAsyncImportStatus'
+     * endpoint.
+     *
+     * @param projectName
+     * @return true/false if its created or not
+     */
+    Boolean createAsyncImportStatusFile(String projectName){
+        try{
+            return asyncImportService.createStatusFile(projectName)
+        }catch(Exception e){
+            e.printStackTrace()
+            throw e
+        }
+    }
+
+    /**
+     * Gets the info of a existing status file in db, if there isn't a status file.
+     *
+     * @param projectName
+     * @return
+     */
+    AsyncImportStatusDTO getAsyncImportStatusFileForProject(String projectName){
+        try{
+            if( asyncImportService.statusFileExists(projectName) ){
+                def dto = asyncImportService.getAsyncImportStatusForProject(projectName)
+                if( !dto || null == dto ) return null
+                return dto
+            }
+        }catch(Exception e){
```

Comment:
this should throw an exception instead of just return null to the client knows that there was an error an should look at the logs instead of just having a "No Status file in db." message